content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -58,12 +58,13 @@ kube-public       Active   1d
 kube-system       Active   1d
 ```
 
-Kubernetes starts with three initial namespaces:
+Kubernetes starts with four initial namespaces:
 
    * `default` The default namespace for objects with no other namespace
    * `kube-system` The namespace for objects created by the Kubernetes system
    * `kube-public` This namespace is created automatically and is readable by all users (including those not authenticated). This namespace is mostly reserved for cluster usage, in case that some resources should be visible and readable publicly throughout the whole cluster. The public aspect of this namespace is only a convention, not a requirement.
-
+   * `kube-node-lease` This namespace for the lease objects associated with each node which improves the performance of the node heartbeats as the cluster scales.
```

Comment:
nit: This namespace is ...

Reply:
Thanks for the review!
Let's move this forward and create a follow-up PR to address the comment.